pymeasure/instruments/lakeshore/lakeshore_base.py

```diff
@@ -45,7 +45,7 @@ class LakeShoreTemperatureChannel(Channel):
     )
     celcius = Instrument.measurement(
```

Comment:
This should be `celsius` as well. But I only touched docstrings so far. What do you think?

Reply:
I think you can fix it, don't worry it will go through a review before merge and it looks completely safe to me.